styles.css

```diff
@@ -2,22 +2,36 @@ html,
 body {
   margin: 0;
   padding: 0;
-  font-family: "Poppins", sans-serif;
+  font-family: 'Poppins', sans-serif;
+  font-size: 16px;
+  line-height: 24px;
   background: #f7f7f9;
 }
 
-header {
+fieldset {
+  margin: 0;
+  padding: 0;
+  border: none;
+}
+
+/* HEADER TOOLBAR SECTION */
+
+#toolbar {
+  box-sizing: border-box;
   position: fixed;
   top: 0;
   left: 0;
   height: 48px;
   width: 100%;
   background: #fff;
   box-shadow: 0 3px 7px rgba(0, 0, 0, 0.1);
+  z-index: 9999;
+  display: flex;
+  justify-content: space-between;
```

Comment:
- [ ] I noticed that you have multiple repeated CSS properties across your stylesheet. It would be great if you can find a way of reducing this duplication of code ♻️. Remember that not repeating pieces of code will keep your codebase clean and readable. 🤓 🛠️

From [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md):
![image](https://user-images.githubusercontent.com/77069011/200000947-9e58c4d8-fed4-45f8-a822-7828c6e713f7.png)

In this case, you could do this with flex:

```CSS
.flex {
  display: flex;
}
```

And then, use it in your HTML elements where you need a flex container:

```HTML
<div class="flex"> This is a flex container </div>
```

Please, consider fixing this! 🤓